src/jobs/fog-status-job.js

```diff
@@ -67,9 +67,15 @@ async function _updateFogStatus (transaction) {
 async function _updateMicroserviceStatus (unknownFogUuids, transaction) {
   const microservices = await MicroserviceManager.findAllWithStatuses({ iofogUuid: unknownFogUuids }, transaction)
   const microserviceStatusIds = microservices
+    .filter((microservice) => microservice.application.isActivated === false)
```

Comment:
Not all microservices will have an application, to avoid any crash, you should do something like `microservice.application && microservice.application.isActivated === false`

Reply:
updated